### PR TITLE
add css-layout

### PR DESCRIPTION
**Pull request by NuriaComa**

### CODE REVIEW

File: staff/NuriaComa/css-layout-simple-adaptive/styles/mobile.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+main{   
+    height: 60%;
+}
+.nav{
+    width: 100%;
+    height: 20%;
+    float: left;
+    border: 1px solid white;
+    box-sizing: border-box;
+}
```
Comment (agandia9) [inline staff/NuriaComa/css-layout-simple-adaptive/styles/mobile.css:31]: Should you use the selector by class when there is not more than one element using that class? investigate the selector by tag. :)

File: staff/NuriaComa/css-layout-simple-adaptive/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/desktop.css">
+    <title>Document</title>
```
Comment (agandia9) [inline staff/NuriaComa/css-layout-simple-adaptive/index.html]: caution default title!

Comment (NuriaComa) [reply staff/NuriaComa/css-layout-simple-adaptive/index.html]: Done :)

File: staff/NuriaComa/css-layout-simple-adaptive/styles/mobile.css (hunk 1 above)
Comment (NuriaComa) [reply staff/NuriaComa/css-layout-simple-adaptive/styles/mobile.css:31]: DONE! thanks!! :)

**Review by agandia9 — COMMENTED** [review]
cool! 😀

**Pull request merged**